### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by VengeUrban**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,381 @@
...
+            transport: 'ipc'
+        })
+        rpc.isConnected = false;
+        rpc.login({
+            'clientId': '750116161890287657'
+        })
+        rpc.isConnected = true;
+        rpc.on("ready", () => {
+            rpc.setActivity({
+                largeImageKey: 'nexi-old',
```
Comment (NeXiDE) [inline main.js:84]: largeImageKey should be changed from nexi-old to nexi-client

File: main.js [hunk 2]
```diff
@@ -1 +1,381 @@
...
+                }
+                else {
+                    if (url.indexOf('index.html') !== -1){
+                        url = rpc.user.username;
+                    }
+                }
+            }
+
+            rpc.setActivity({
+                largeImageKey: 'nexi-old',
```
Comment (NeXiDE) [inline main.js:119]: largeImageKey should be changed from nexi-old to nexi-client

Comment (VengeUrban) [reply main.js:119]: done

**Review by NeXiDE — APPROVED** [review]
Perfect.

**Pull request merged**